handbook/company/communications.md

```diff
@@ -213,31 +213,21 @@ Fleet prefixes all GitHub labels with special characters or words to organize an
 
 | Prefix            | Label type  | Examples                            |
 |:------------------|:------------|:------------------------------------|
-| `#`               | Noun        | `#g-marketing`, `#g-ceo`, `#agent`
```

Comment:
I propose we remove the noun label altogether because it creates inconsistency. Many labels can be either nouns or adjectives. Because labels are metadata, we want them to describe the issue. In this way, an `~agent` label describes the issue and is used as an adjective vs. a noun.

There were only a few labels using `#` so in practice a small change.

Reply:
Note the line further down documents the `#g-` usage.